etc/spack/defaults/modules.yaml

```diff
@@ -43,3 +43,17 @@ modules:
   lmod:
     hierarchy:
       - mpi
+  tcl:
```

Comment:
question: add same block to lmod or is there a "catch-all" we can use?

Reply:
I don't think this modification is needed as long as we have the same variable set in `setup_environment`.

Reply:
thanks, i'll just remove this then

Reply:
cannot update the PR anymore.